apps/studio/src/server/modules/page/page.router.ts

```diff
@@ -96,4 +99,22 @@ export const pageRouter = router({
       return { pageId: "" }
     }),
   // TODO: Delete page stuff here
+
+  readImageInPage: pageProcedure
```

Comment:
unsure why this needs to be processed on the server. If there is already a URL, shouldn't the client be able to render it? are we using the server as a means to mask the public image URL? suspect this would be very taxing on the server.